src/libraries/IterableMap.sol

```diff
@@ -0,0 +1,104 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/Controller.sol';
+
+contract IterableMap {
+
+    struct Item {
+        bool hasItem;
+        uint256 value;
+        uint256 offset;
+    }
+
+    address owner;
+    bool initialized;
+    uint256[] itemsArray;
+    Item[] itemsMap;
+    uint numberOfItems;
+
+    function any() {
```

Comment:
Does Solidity have any() functions that work like Serpent's any() functions (i.e., they are executed before all functions except init())?  If not, this will need to be done differently.